src/Bindle/BindleClient.cs

```diff
@@ -15,34 +15,38 @@ namespace Deislabs.Bindle;
 public class BindleClient
 {
     public BindleClient(
-        string baseUri
+        string connectionString
     )
     {
-        _baseUri = new Uri(SlashSafe(baseUri));
-        _httpClient = new HttpClient();
-    }
+        ConnectionInfo connectionInfo = new ConnectionInfo(connectionString);
 
-    public BindleClient(
-        string baseUri,
-        HttpMessageHandler messageHandler
-    )
-    {
-        _baseUri = new Uri(SlashSafe(baseUri));
-        _httpClient = new HttpClient(messageHandler);
+        // TODO: do we want to assume the default listening address?
```

Comment:
Do we have any defaults for similar uri/url values elsewhere?  First candidate might be `localhost:<port>`... that being said, I think I like err-ing on the side of no defaults and hard error rather than a potentially-hidden default leading to perhaps not as obvious failure mode behavior.

Reply:
Thanks for the validation! Will remove the TODO